experiment/run_experiment.py

```diff
@@ -332,6 +385,16 @@ def filter_file(tar_info):
         for benchmark in config['benchmarks']:
             add_oss_fuzz_corpus(benchmark, oss_fuzz_corpora_dir)
 
+    if config['random_seed_corpus_dir']:
+        for benchmark in config['benchmarks']:
+            benchmark_corpus_archive_path = os.path.join(
+                config['random_seed_corpus_dir'], f'{benchmark}.zip')
+            filestore_utils.cp(
+                benchmark_corpus_archive_path,
+                experiment_utils.get_random_seed_corpora_filestore_path() + "/",
```

Comment:
Should this use `join`?

Reply:
Just want to align with the code above where they used this path concat style: https://github.com/Practical-Formal-Methods/fuzzbench-public/blob/5b9cc505d4c661430a03770c0f3c5e60d277f2ce/experiment/run_experiment.py#L373

Should I change it?

Reply:
Ah, I hadn't seen that. No, then keep it like that. Thanks!